src/app/common/constants.ts

```diff
@@ -12,6 +12,20 @@ export enum ProviderType {
   openshift = 'openshift',
 }
 
+export enum PlanStatusConditionsType {
+  ready = 'Ready',
+  running = 'Running',
+  finished = 'Finished',
+  error = 'Error',
+}
+
+export enum PlanStatusType {
+  ready = 'Ready',
+  running = 'Running',
+  finished = 'Complete',
+  error = 'Failed',
+}
```

Comment:
Is the value in the actual API capitalized or no? For clarity we may want to use the same exact string here for the key and value, so either `Ready = 'Ready'` or `ready = 'ready'`